DSPs/008-Token-Redemption.md

```diff
@@ -0,0 +1,21 @@
+# DORG Token Redemption
+
+**Author(s)**: @nestorbe
+
+## Summary
+
+Build the UI Client for the dOrg token redemption/staking contracts.
+
+## Why should we fund this?
+
+This project will give usability to the current dOrg token deployed contract. It will let the token holder redeem it for a choice of the preferred stablecoin, it will also enable the ability to stake the token to earn an APR based on the amount of dOrg token the holder commits to staking.
```

Comment:
Hey @nestorbe , do you plan on deploying this UI and setting up DNS for it? Like maybe token.dorg.tech? That way it would give usability to the dOrg token contract in a truly production environment.

Reply:
Hey @benefacto ! yes, after we discuss and select the APR amount for the StakingReward contract, that would be the next step, deploying and yes, maybe setting up a DNS for it (this we haven't discussed yet), but that sounds good 'token.dorg.tech'.